### PR TITLE
[config reload]Modifying config reload tests to align with latest config reload enhancement

### PR DESCRIPTION
**Pull request by dgsudharsan**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
Modifying config reload tests to align with latest config reload enhancement according to HLD https://github.com/sonic-net/SONiC/pull/1203

#### How did you do it?
Removed the timer based checks

#### How did you verify/test it?
Run the tests with config reload enhamcenet changes

### CODE REVIEW

File: tests/common/config_reload.py [hunk 1]
```diff
@@ -43,16 +43,6 @@ def config_system_checks_passed(duthost):
         if int(out['stdout'].strip()) < 120:
             return False
 
-    logging.info("Checking if delayed services are up")
```
Comment (qiluo-msft) [inline tests/common/config_reload.py:46]: >delayed [](http://example.com/codeflow?start=30&length=7)

Do you have new ways to check delayed services?

Comment (dgsudharsan) [reply tests/common/config_reload.py:46]: @qiluo-msft No. But the waiting for swss for 120 seconds should cover this as well.

Comment (qiluo-msft) [reply tests/common/config_reload.py:46]: Originally the test verify lldp/pmon/snmp/telemetry/etc started. I do not understand "waiting for swss for 120 seconds should cover this".

Comment (dgsudharsan) [reply tests/common/config_reload.py:46]: Previously all these services took 3 min 30 seconds to boot up and hence it is greater than 120 seconds wait for swss. Now these services almost within 30 seconds of switch initialization (at the worst a min). So essentially config reload will not proceed for another 60 seconds which is quite sufficient for these services to start.
Hence not adding any checks here.

**Pull request merged**